docs/reference/central-configuration.md

```diff
@@ -0,0 +1,76 @@
+---
+navigation_title: Central configuration
+description: Reference documentation for the central configuration of EDOT SDKs.
+applies_to:
+  deployment:
+      ess: preview 9.1
+  stack: preview 9.1
+products:
+  - id: cloud
+  - id: observability
+  - id: kibana
+---
+
+# Central configuration for EDOT SDKs
+
+You can manage {{edot}} (EDOT) SDKs through the APM central configuration feature in the Applications UI. Changes are automatically propagated to the deployed [EDOT SDKs](./edot-sdks/index.md). Refer to [APM central configuration](docs-content://solutions/observability/apm/apm-agent-central-configuration.md) for more information.
+
+## Prerequisites
+
+To use central configuration for EDOT SDKs, you need:
+
+* An Elastic self-managed or {{ecloud}} deployment, version 9.1 or higher.
+* [EDOT Collector](./edot-collector/index.md) in standalone mode.
+* [EDOT SDKs](./edot-sdks/index.md) instrumenting your application.
+
+The following versions of EDOT support central configuration:
+
+| Component | Minimum Version |
+|-----------|----------------|
+| EDOT Collector | 9.1 / 8.19 or higher |
```

Comment:
It took me a few reads to figure out why you don't just say 8.19 or higher. I guess because 9.0 isn't supported? Would `8.19, 9.1 or higher` make more sense?

Reply:
Yeah, I wasn't happy with this one either. I like your suggestion!